source/edge-server/configure.txt

```diff
@@ -18,113 +18,94 @@ Configure Edge Server
-         the Atlas UI, choose :guilabel:`Real-time Sync`.
-         
-         These templates set up a Device Sync-enabled task tracker app with a 
-         downloadable front-end client. The template app comes pre-configured 
-         with permissions for the user to write their own Items to an ``Item`` 
-         collection in a ``todo`` database. It also has 
-         :ref:`Development Mode <development-mode>` and 
-         :ref:`email/password authentication <email-password-authentication>` 
-         enabled.
+The host that runs Edge Server instance may be an EC2 instance or other cloud 
+server, or hardware that you provision on-site.
 
-         On the :ref:`edge-server-connect` page, you can learn how to 
-         modify the client to connect to the Edge Server.
+.. tip::
 
-   .. step:: Enable Device Sync
+   For proof-of-concept testing, you can launch an EC2 instance running 
+   Ubuntu Server 18.x. Under Network settings, add a rule to allow access 
+   via HTTP/Port 80. Free tier settings are sufficient for small PoC projects.
 
-      If you have not started with a template app, :ref:`enable Device Sync 
-      <enable-sync>` with Flexible Sync. 
-      
-   .. step:: Configure Data Access Permissions
+.. _edge-server-hardware-requirements:
 
-      Configure any :ref:`permissions <device-sync-compatibility>` your 
-      App needs to enforce data access rules.
+Required Hardware
+~~~~~~~~~~~~~~~~~
 
-   .. step:: Configure an Authentication Provider
+The current minimum hardware requirements to run an Edge Server instance are:
 
-      Edge Server supports these authentication providers:
+- CPU: 64-bit, quad-core processor
+- RAM: 8GB memory
+- Storage: 32GB disk space
 
-      - :ref:`Anonymous Authentication <anonymous-authentication>`
-      - :ref:`Email/Password Authentication <email-password-authentication>`
-      - :ref:`Custom JWT Authentication <custom-jwt-authentication>`
-      - :ref:`API Key Authentication <api-key-authentication>`
+These requirements may change after Edge Server benchmarking is complete.
 
-      Enable an authentication provider for clients to connect to the Edge Server.
+.. _edge-server-os-support:
 
-      .. note::
-         
-         If you created a :ref:`template app <template-apps>`, email/password 
-         authentication is already enabled in the template app configuration.
+OS Compatibility
+~~~~~~~~~~~~~~~~
 
-   .. step:: Get Edge Server Enabled
+Edge Server has been verified with the following operating systems:
 
-      At this time, Edge Server is in Private Preview. Your MongoDB 
-      representative must enable Edge Server for your App. Give
-      your :ref:`App Services App ID <find-app-id>` to your 
-      Product or Account Representative. They will enable Edge Server and 
-      generate an authorization secret for your App.
+- Ubuntu Server 18+
+- macOS 11.x and newer running on M1 or Intel 2018+ MacBook Pros
```

Comment:
you can add raspberry pi os 12 (https://www.raspberrypi.com/software/operating-systems/)